### PR TITLE
Changing filters for vulnerability view

### PR DESCRIPTION
**Pull request by akuleshov7**
STEP 1: We will need to add A LOT of filters on this page, but now we should at least refactor and split filters for it

### What's done:
- updating filters
- polishing vuln. collection view

image

### CODE REVIEW

**Comment by akuleshov7** [comment]
We need to have a possibility to filter criticality: https://mdbootstrap.com/docs/b4/jquery/plugins/multi-range/

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/table/filters/NameFiltersRow.kt [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                }
+            }
+            button {
+                type = ButtonType.button
+                className = ClassName("btn btn-secondary mr-3")
+                fontAwesomeIcon(icon = faSearch)
+                onClick = {
+                    props.onChangeFilters(filtersName)
+                }
+            }
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/table/filters/NameFiltersRow.kt:60]:
```suggestion
            buttonBuilder(faSearch, style = "secondary", classes = "mr-3") {
                props.onChangeFilters(filtersName)
            }
```

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/table/filters/NameFiltersRow.kt [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            button {
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/table/filters/NameFiltersRow.kt:61]: same here

**Review by sanyavertolet — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/table/filters/NameFiltersRow.kt (hunk 1 above)
Comment (akuleshov7) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/table/filters/NameFiltersRow.kt:60]: If you can see - I have updated new filters in new code to button builder, this is old one...
I will upgrade ALL buttons to buttonBuilder a little bit later in one PR

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/table/filters/NameFiltersRow.kt (hunk 2 above)
Comment (akuleshov7) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/table/filters/NameFiltersRow.kt:61]: yep, same in TestExecutionFilters

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/table/filters/NameFiltersRow.kt (hunk 1 above)
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/table/filters/NameFiltersRow.kt:60]: oh ok got it

**Review by 0x6675636b796f75676974687562 — APPROVED** [review]

**Review by Cheshiriks — APPROVED** [review]

**Pull request merged**